README.md

```diff
@@ -153,6 +154,15 @@ For Puppet 4.2.2+ (and 3.8.2) we've lessened the possibility of the issue occurr
 
 If you are affected by this issue, a known workaround is to download the `.tar.gz` from the forge and use `puppet module install` using the file rather than directly from the forge.
 
+### Running Puppet and DSC without Administrative Privileges
+
+While there are avenues for using Puppet with a non-administrative account, DSC is limited to only accounts with administrative privileges.
```

Comment:
Whoops, your second sentence was fine. Looks like you had the edit knife in here a bit much. ;)